### PR TITLE
Dịch "mlp_vn" - Phần 6

### LINKED ISSUE #831: Dịch "mlp_vn" - Phần 6
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/mlp_vn.md#L375-L450

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by lamngoctam**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #831

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 1]
```diff
@@ -410,14 +412,19 @@ However, the sigmoid has mostly been replaced by the simpler and more easily tra
 In the "Recurrent Neural Network" chapter (:numref:`sec_plain_rnn`), we will describe architectures that leverage sigmoid units to control the flow of information across time.
 -->
 
-*dịch đoạn phía trên*
+Khi sự chú ý chuyển sang gradient dựa trên sự học, hàm sigmoid là một lựa chọn tất yếu bởi vì nó có tính xấp xỉ liên tục và khả vi đối với một đơn vị ngưỡng.
+Sigmoids vẫn được sử dụng rộng rãi như các hàm kích hoạt trên các đơn vị đầu ra,
+khi chúng tôi muốn biểu diễn các kết quả đầu ra như các xác suất cho các bài toán phân loại nhị phân (bạn có xem sigmoid như một trường hợp đặc biệt của softmax).
+Tuy nhiên, hầu hết sigmoid đã được thay thế bằng ReLU bởi sự đơn giản và dễ huấn luyện hơn để sử dụng trong các tầng ẩn.
+Trong chương "Mạng nơ-ron truy hồi" (:numref:`sec_plain_rnn`), chúng tôi sẽ mô tả các kiến trúc sử dụng các đơn vị sigmoid để kiểm soát luồng thông tin theo thời gian.
```
Comment (lamngoctam) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Trong chương "Mạng nơ-ron truy hồi" (:numref:`sec_plain_rnn`), chúng tôi sẽ mô tả các mô hình sử dụng các đơn vị sigmoid để kiểm soát luồng thông tin theo thời gian.
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 2]
```diff
@@ -382,14 +382,15 @@ This makes optimization better behaved and it mitigated well-documented problem
...
+### Hàm Sigmoid
 
 <!--
 The sigmoid function transforms its inputs, which values in the domain $\mathbb{R}$, to outputs that lie the interval $(0, 1)$.
 For that reason, the sigmoid is often called a *squashing* function: it *squashes* any input in the range (-inf, inf) to some value in the range (0, 1).
 -->
 
-*dịch đoạn phía trên*
+Hàm sigmoid biến đổi các biến đầu vào của nó có giá trị nằm trong miền $\mathbb{R}$, để có được các giá trị đầu ra nằm trong khoảng $(0, 1)$.
+Với sự biến đổi như trên, hàm sigmoid thường được gọi là hàm *nén*: có nghĩa là *nén* bất kỳ biến đầu vào nào nằm trong khoảng (vô cùng bé, vô cùng lớn) để có được giá trị đầu ra nằm trong khoảng (0, 1).
```
Comment (lamngoctam) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Với sự biến đổi như trên, hàm sigmoid thường được gọi là hàm *nén*: có nghĩa là *nén* bất kỳ biến đầu vào nằm trong khoảng (vô cùng bé, vô cùng lớn) để có được giá trị đầu ra nằm trong khoảng (0, 1).
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 3]
```diff
@@ -440,7 +447,9 @@ Note that when the input is 0, the derivative of the sigmoid function reaches a
 As the input diverges from 0 in either direction, the derivative approaches 0.
 -->
 
-*dịch đoạn phía trên*
+Đạo hàm của hàm sigmoid được vẽ sau đây.
+Chú ý rằng khi biến đầu vào là 0, đạo hàm của hàm sigmoid đạt giá trị đối đa là 0.25.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Chú ý rằng khi đầu vào là 0, đạo hàm của hàm sigmoid đạt giá trị lớn nhất là 0.25.
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 4]
```diff
@@ -440,7 +447,9 @@ Note that when the input is 0, the derivative of the sigmoid function reaches a
...
+Vì biến đầu vào phân kỳ từ 0 theo một trong hai hướng, đạo hàm sẽ đạt giá trị là 0.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Khi đầu vào phân kỳ từ 0 theo một trong hai hướng, đạo hàm sẽ tiến tới 0.
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 5]
```diff
@@ -584,7 +593,7 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 5 -->
-*
+* Lâm Ngọc Tâm
 
 <!-- Phần 6 -->
 *
```
Comment (lamngoctam) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
* Lâm Ngọc Tâm
* Phạm Minh Đức
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 6]
```diff
@@ -399,8 +400,9 @@ Thus the pioneers of this field, going all the way back to McCulloch and Pitts,
 A thresholding activation takes value $0$ when its input is below some threshold and value $1$ when the input exceeds the threshold.
 -->
 
-*dịch đoạn phía trên*
-
+Từ những nghiên cứu đầu tiên về mạng nơ-ron, các nhà khoa học đã quan tâm đến việc mô hình hóa các nơ-ron sinh học ở hướng *kích hoạt* hoặc *không kích hoạt*.
+Vì vậy mà những người tiên phong trong lĩnh vực này gồm McCulloch và Pitts, những nhà phát minh ra nơ-ron nhân tạo, sẽ tập trung nghiên cứu vào các đơn vị ngưỡng.
```
[the remark this answers is not in the document]
Comment (lamngoctam) [reply chapter_multilayer-perceptrons/mlp_vn.md]: @minhduc0711 Cảm ơn sự góp ý của bạn. Vậy để xem thêm vài review về chữ `unit` trong ngữ cảnh này ntn.

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 7]
```diff
@@ -410,14 +412,19 @@ However, the sigmoid has mostly been replaced by the simpler and more easily tra
 In the "Recurrent Neural Network" chapter (:numref:`sec_plain_rnn`), we will describe architectures that leverage sigmoid units to control the flow of information across time.
 -->
 
-*dịch đoạn phía trên*
+Khi sự chú ý chuyển sang gradient dựa trên sự học, hàm sigmoid là một lựa chọn tất yếu bởi vì nó có tính xấp xỉ liên tục và khả vi đối với một đơn vị ngưỡng.
```
[the remark this answers is not in the document]
Comment (lamngoctam) [reply chapter_multilayer-perceptrons/mlp_vn.md]: @minhduc0711 Đoạn `Khi phương pháp học dựa trên gradient trở nên thu hút,` bạn dịch hợp lý. Còn đoạn sau mình muốn xem thêm các review khác ntn.

[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/mlp_vn.md]: @lamngoctam mình mới sửa lại cho mượt, hôm qua buồn ngủ nên dịch hơi linh tinh :sweat_smile:

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 8]
```diff
@@ -399,8 +400,9 @@ Thus the pioneers of this field, going all the way back to McCulloch and Pitts,
 A thresholding activation takes value $0$ when its input is below some threshold and value $1$ when the input exceeds the threshold.
 -->
 
-*dịch đoạn phía trên*
-
+Từ những nghiên cứu đầu tiên về mạng nơ-ron, các nhà khoa học đã quan tâm đến việc mô hình hóa các nơ-ron sinh học, thứ có thể ở trong trạng thái *kích hoạt* hoặc *không kích hoạt*.
+Vì vậy mà những người tiên phong trong lĩnh vực này, bao gồm McCulloch và Pitts, những người phát minh ra nơ-ron nhân tạo, tập trung nghiên cứu các đơn vị ngưỡng.
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Vì vậy mà những người tiên phong trong lĩnh vực này, bao gồm [McCulloch](https://en.wikipedia.org/wiki/Warren_Sturgis_McCulloch) và [Pitts](https://en.wikipedia.org/wiki/Walter_Pitts), những người phát minh ra nơ-ron nhân tạo, tập trung nghiên cứu các đơn vị ngưỡng.
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 9]
```diff
@@ -410,14 +412,19 @@ However, the sigmoid has mostly been replaced by the simpler and more easily tra
 In the "Recurrent Neural Network" chapter (:numref:`sec_plain_rnn`), we will describe architectures that leverage sigmoid units to control the flow of information across time.
 -->
 
-*dịch đoạn phía trên*
+Khi sự chú ý chuyển sang gradient dựa trên sự học, hàm sigmoid là một lựa chọn tất yếu bởi vì nó có tính xấp xỉ liên tục và khả vi đối với một đơn vị ngưỡng.
+Hàm sigmoid vẫn là hàm kích hoạt được sử dụng rộng rãi ở các đơn vị đầu ra,
+khi ta muốn biểu diễn kết quả đầu ra như các xác suất cho bài toán phân loại nhị phân (bạn có thể xem sigmoid như một trường hợp đặc biệt của softmax).
+Tuy nhiên, trong các tầng ẩn, hàm sigmoid đã gần như bị thay thế bằng hàm ReLU vì nó đơn giản hơn và khiến cho việc huấn luyện trở nên dễ dàng hơn.
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/mlp_vn.md]: Từ `khiến` này nghe.... negative anh ạ...
```suggestion
Tuy nhiên, trong các tầng ẩn, hàm sigmoid đã gần như bị thay thế bằng hàm ReLU vì nó đơn giản hơn và giúp cho việc huấn luyện trở nên dễ dàng hơn.
```

File: chapter_multilayer-perceptrons/mlp_vn.md (hunk 8 above)
Comment (lamngoctam) [reply chapter_multilayer-perceptrons/mlp_vn.md]: @duythanhvn à được thêm chú thích về scientists nữa mà mình ko hay.

Comment (duythanhvn) [reply chapter_multilayer-perceptrons/mlp_vn.md]: Dạ tuỳ chỗ ấy anh, tại em thấy để tên hai nhân vật này đã không đầy đủ mà còn không trích dẫn có khi người đọc cũng không biết họ là ai...

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 10]
```diff
@@ -382,14 +382,15 @@ This makes optimization better behaved and it mitigated well-documented problem
...
-### *dịch tiêu đề phía trên*
+### Hàm Sigmoid
 
 <!--
 The sigmoid function transforms its inputs, which values in the domain $\mathbb{R}$, to outputs that lie the interval $(0, 1)$.
 For that reason, the sigmoid is often called a *squashing* function: it *squashes* any input in the range (-inf, inf) to some value in the range (0, 1).
 -->
 
-*dịch đoạn phía trên*
+Hàm sigmoid biến đổi các giá trị đầu vào thuộc $\mathbb{R}$ thành các giá trị đầu ra nằm trong khoảng $(0, 1)$.
```
Comment (rootonchair) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Hàm sigmoid biến đổi các giá trị đầu vào có miền giá trị thuộc $\mathbb{R}$ thành các giá trị đầu ra nằm trong khoảng $(0, 1)$.
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 11]
```diff
@@ -399,8 +400,9 @@ Thus the pioneers of this field, going all the way back to McCulloch and Pitts,
 A thresholding activation takes value $0$ when its input is below some threshold and value $1$ when the input exceeds the threshold.
 -->
 
-*dịch đoạn phía trên*
-
+Từ những nghiên cứu đầu tiên về mạng nơ-ron, các nhà khoa học đã quan tâm đến việc mô hình hóa các nơ-ron sinh học, thứ có thể ở trong trạng thái *kích hoạt* hoặc *không kích hoạt*.
```
Comment (rootonchair) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
Từ những nghiên cứu đầu tiên về mạng nơ-ron, các nhà khoa học đã quan tâm đến việc mô hình hóa các nơ-ron sinh học, thứ có thể ở một trong hai trạng thái *kích hoạt* hoặc *không kích hoạt*.
```

File: chapter_multilayer-perceptrons/mlp_vn.md [hunk 12]
```diff
@@ -587,7 +596,8 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 6 -->
-*
+* Lâm Ngọc Tâm
+* Phạm Minh Đức
```
Comment (lamngoctam) [inline chapter_multilayer-perceptrons/mlp_vn.md]:
```suggestion
* Phạm Minh Đức
* Phạm Hồng Vinh
```
